wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -176,6 +182,76 @@ public Response create(
     return Response.status(201).entity(asDtoWithLinksAndToken(workspace)).build();
   }
 
+  @Beta
+  @Path("/devfile")
+  @POST
+  @Consumes(APPLICATION_JSON)
+  @Produces(APPLICATION_JSON)
+  @ApiOperation(
+      value = "Creates a new workspace based on the Devfile.",
+      notes =
+          "This method is in beta phase. It's strongly recommended to use `DevfileService#createFromYaml` "
```

Comment:
`DevfileService#createFromYaml` doesn't say much to the reader of the swagger docs. Maybe replace it with the actual endpoint path?

Reply:
Since it's swagger annotation what will be a better reference here? WDYT?
`POST /devfile`, or `#!/devfile/createFromYaml`
![Screenshot_20190424_113250](https://user-images.githubusercontent.com/5887312/56644633-bb8ff200-6684-11e9-97aa-70ac07d9dffa.png)

Reply:
That would depend on how `#!/devfile/createFromYaml` would be rendered. If it became a link then that would be awesome, otherwise I'd be for `POST /devfile`.

Reply:
changed to `POST /devfile`